ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse.yaml

```diff
@@ -33,13 +33,20 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
+- as: installer-rehearse-aws-capi
+  interval: 960h
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+    workflow: cucushift-installer-rehearse-aws-ipi-capi
 - as: installer-rehearse-aws-qe
   interval: 960h
   steps:
     cluster_profile: aws-qe
     env:
       BASE_DOMAIN: qe.devcluster.openshift.com
-    workflow: cucushift-installer-rehearse-aws-ipi-ovn-fips
+    workflow: cucushift-installer-rehearse-aws-ipi-ovn-sdn2ovn
```

Comment:
Does this need to be modified? Because we modified two places in this yaml, so the rehearse is triggered twice. cc @jianlinliu

Reply:
If want to test the new workflow, users can directly replace the workflow here, not need to add new items (from line 36 to 42)

Reply:
Thanks. @miyadav we can remove line 36~42, as they are introducing an additional test which makes the execution slower.

Reply:
And update this line to the new workflow `cucushift-installer-rehearse-aws-ipi-capi`

Reply:
Adding more info on it , this is some generated metadata when I ran - `make registry-metadata `as the test was failing with this [issue](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/29211/pull-ci-openshift-release-master-step-registry-metadata/1534786750797320192)

Reply:
Hi @miyadav , `make` will not touch this file, the file is under our own control. `make` it will generate the `.metadata.json` and update the `ci-operator/jobs/openshift/verification-tests/openshift-verification-tests-master-periodics.yaml`.